src/modeler/AutoRest.Swagger/Validation/TrackedResourcePatchOperationValidation.cs

```diff
@@ -38,24 +38,28 @@ public class TrackedResourcePatchOperationValidation : TypedRule<Dictionary<stri
         /// </summary>
         public override Category Severity => Category.Error;
 
-        // Verifies if a tracked resource has a corresponding PATCH operation
```

Comment:
Don't think you need to do anything (apart from probably adding specifics to the FileObjectPath)